common/src/main/java/io/netty/util/ConstantPool.java

```diff
@@ -31,6 +31,7 @@
 
     private final ConcurrentMap<String, T> constants = PlatformDependent.newConcurrentHashMap();
 
+    @Deprecated
```

Comment:
@normanmaurer it is used for public method `nextId()`. As we don't know how it could be used so I leave it for back compatibility.